index.html

```diff
@@ -0,0 +1,236 @@
+              <p class="browse__cousine browse__item-sub ">ITALIAN</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ ">
+            <div class="browse__2 browse__item ">
+              <p class="browse__recipes browse__item-sub ">856 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">INDIAN</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ ">
+            <div class="browse__3 browse__item ">
+              <p class="browse__recipes browse__item-sub ">27 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">FRENCH</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ ">
+            <div class="browse__4 browse__item ">
+              <p class="browse__recipes browse__item-sub ">174 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">STEAKHOUSE</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ ">
+            <div class="browse__5 browse__item ">
+              <p class="browse__recipes browse__item-sub ">731 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">SEAFOOD</p>
+            </div>
+          </a>
+        </div>
+
+        <div class="browse__container-test ">
+          <a class="browse__link " href="/ ">
+            <div class="browse__6 browse__item ">
+              <p class="browse__recipes browse__item-sub ">237 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">SUSHI</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ ">
+            <div class="browse__7 browse__item ">
+              <p class="browse__recipes browse__item-sub ">529 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">MEXICAN</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ ">
+            <div class="browse__8 browse__item ">
+              <p class="browse__recipes browse__item-sub ">145 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">CHINESE</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ ">
+            <div class="browse__9 browse__item ">
+              <p class="browse__recipes browse__item-sub ">327 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">PIZZA</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ ">
+            <div class="browse__10 browse__item ">
+              <p class="browse__recipes browse__item-sub ">1.437 Recipes</p>
+              <p class="browse__cousine browse__item-sub ">AMERICAN</p>
+            </div>
+          </a>
+          <a class="browse__link " href="/ "></a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="statistic ">
+    <div class="container statistic__content ">
+      <div class="statistic__item ">
```

Comment:
why extra space at the end of `class` Could you please fix such cases?

Reply:
Fixed. Thank you for noticing.